### PR TITLE
Add `osu! Medal Hunters Discord server` article

### PR DESCRIPTION
**Pull request by minusQuantumNeko**
Related to Issue #6904

## Self-check

- [X] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Review by sharytory — DISMISSED** [review]
very goog

File: wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md
Comment (TicClick) [inline wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md]: doesn't deserve a tag, too broad

File: wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md
Comment (TicClick) [inline wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md]: not so sensational
```suggestion
New members can send messages on server by default, but in order to be able to send images and have medals percentage next to their nickname they have to authenticate using authentication link sent by server private bot, Osekai Authenticator.

![Authentication Message Example](img/authentication_example.png "An example of bot authentication message when you join server") ![Medals Percentage Example](img/medals_percentage.png "An example of medals percentage next to user nickname")

Members can't change their server nickname, so this percentage is showing their real medals completion progress.
```

File: wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md
Comment (TicClick) [inline wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md]: I guess I can live with this many exclamations, but doesn't this read as a repetition of "it's about medals, and medal hunting" over and over, in different ways?

**Review by TicClick — CHANGES_REQUESTED** [review]

File: wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md
Comment (TicClick) [inline wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md:17]: I don't think you need this

File: wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md
Comment (TicClick) [inline wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md:28]: mixed feelings due to https://github.com/ppy/osu-wiki/pull/5736#issuecomment-866580536

Comment (Walavouchey) [reply wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md:28]: more mixed feelings due to https://github.com/ppy/osu-wiki/pull/7949#discussion_r962355297

File: wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md
Comment (Walavouchey) [reply wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md:17]: was copied from https://github.com/ppy/osu-wiki/issues/5440, and i think you do. at least https://markdownlivepreview.com/ doesn't consider it valid otherwise

Comment (TicClick) [reply wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md:17]: I misread the table, thinking the alignment is all you need. yeah totally, carry on

File: wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md
Comment (Walavouchey) [reply wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md:28]: i'm gonna require @Ephemeralis's opinion here. i put a warning at the top of the page and i don't know exactly how more gracefully i should word this.

Comment (TicClick) [reply wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md:28]: I guess we can let it pass since a) it doesn't mention the medals which shall not be named, and b) it's an open secret anyway, but I'd still run it past @peppy just to get an explicit approval/rejection

**Review by TicClick — DISMISSED** [review]
looks good to me except for a comment in https://github.com/ppy/osu-wiki/pull/8152#discussion_r990449430 which needs second opinion

**Review by Walavouchey — DISMISSED** [review]

File: wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md
Comment (Walavouchey) [reply wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md:28]: a) well sorta

again, as i said i put a very explicit notice at the top of the page

> *Notice: While working together in order to find Hush-Hush medals is encouraged, finding them by yourself is as well. Be advised that the website and server contain solutions to Hush-Hush medals.*

File: wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md
Comment (Hiviexd) [inline wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md:37]:
```suggestion
| Osekai developer & server admin | ::{ flag=GB }:: [Tanza3D](https://osu.ppy.sh/users/10379965) (Hubz#6283, Tanza#8733), ::{ flag=DE }:: [mulraf](https://osu.ppy.sh/users/1309242) (mulraf#1087) |
```

File: wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md
Comment (Hiviexd) [inline wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md:38]:
```suggestion
| Osekai community manager & server admin | ::{ flag=UA }:: [MegaMix_Craft](https://osu.ppy.sh/users/18152711) (MegaMix_Craft#1151), ::{ flag=GB }:: [chromb](https://osu.ppy.sh/users/10238680) (chromb#4528), ::{ flag=CA }:: [Remyria](https://osu.ppy.sh/users/1699875) (Remyria#0001) |
```

File: wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md
Comment (Hiviexd) [inline wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md:39]: missing flag

**Review by Hiviexd — CHANGES_REQUESTED** [review]

File: wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md
Comment (minusQuantumNeko) [reply wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md:38]: Hey, @Hiviexd, I'm not from GB, so country code is actually correct for this one!

File: wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md
Comment (minusQuantumNeko) [inline wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md:24]: I think it's fine like that, but just wanted to clarify that the only place on server you can find solutions at is `#announcements` channel, and only for recent medals.

Comment (Tanza3D) [reply wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md:24]: you're really likely to run into them in bot commands and just generally across the server, as bathbot and other people may post solutions. best to cover the entire server with that I feel

Comment (minusQuantumNeko) [reply wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md:24]: Hmm, makes sense!

File: wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md
Comment (peppy) [reply wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md:28]: No.

In fact I'm not sure this article should exist at all? Are we suddenly allowing third party discords to advertise on the wiki?

Comment (Walavouchey) [reply wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md:28]: assuming that only concerns this article (as it has a reason), since some other servers also have closely related websites (e.g. osu! storyboarder banquet with storyboarder.xyz, and osu! University with its youtube, spotify and twitter etc.)

Comment (peppy) [reply wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md:28]: I dunno. Highlight me before linking external sites and we'll handle on a per-case basis.

Comment (TicClick) [reply wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md]: I went ahead and removed the website mentions [`9ab273d` (#8152)](https://github.com/ppy/osu-wiki/pull/8152/commits/9ab273d9bb7901a39c05ef10b2f3a43c40705c14), hope that works better

**Review by TicClick — APPROVED** [review]

**Pull request merged**
